spec/classes/config_spec.rb

```diff
@@ -77,6 +77,18 @@
     }
   end
 
+  context 'log4j file' do
+    log4j_file = '/tmp/log4j.properties'
+    File.open(log4j_file, "w") { |file| file.write("log4j.rootLogger=INFO, CONSOLE") }
```

Comment:
Otherwise I will have to remove that test. Can I just create the empty file instead?